exercises/triangle/test/test_triangle.c

```diff
@@ -4,101 +4,97 @@
 void test_equilateral_is_true_if_all_sides_are_equal(void)
 {
    Triangle_t sides = { 2, 2, 2 };
-   TEST_ASSERT(is_equilateral(&sides));
+   TEST_ASSERT_TRUE(is_equilateral(&sides ));
 }
 
 void test_equilateral_is_false_if_any_side_is_unequal(void)
 {
    Triangle_t sides = { 2, 3, 2 };
-   TEST_ASSERT(!(is_equilateral(&sides)));
+   TEST_ASSERT_FALSE(is_equilateral(&sides));
 }
 
 void test_all_zero_sides_are_illegal_so_the_triangle_is_not_equilateral(void)
 {
    Triangle_t sides = { 0, 0, 0 };
-   TEST_ASSERT(!(is_equilateral(&sides)));
+    TEST_ASSERT_FALSE(is_equilateral(&sides ));
 }
 
 void test_in_equilateral_sides_may_be_floats(void)
```

Comment:
Remove `_in` here. Leave as `test_equilateral_sides_may_be_floats`, as it makes more sense